src/Plugin/Field/FieldFormatter/StrawberryPagedFormatter.php

```diff
@@ -371,6 +371,7 @@ public function processElementforMetadatadisplays($delta, array $jsondata, Field
     $element = [];
     $entity = NULL;
     $nodeuuid = $item->getEntity()->uuid();
+    $has_ocr = $this->searchEnabled($item);
```

Comment:
We are not using this anymore. If we use this flag then it will be cached. That means once OCR is available it won't show

Reply:
Ok, I understand it, so to enable/disable text selection if no ocr we have to make a sync ajax call before setting options in js, right?

Reply:
Yes, we do that in https://github.com/esmero/format_strawberryfield/blob/66bbd57e9d90982e46bb925d49640693aba27dc9/js/iiif-iabookreader_strawberry.js#L40-L50
So for your use case we need to do it before so you can pass the argument to the `br`.

Means that function can return has_ocr (which can start being FALSE always).  Does that work?

Reply:
@DiegoPino I made that before, it works but I received a console deprecation because of ajax sync in main thread so I switched to this last solution, bad, so I can pull my first code whit deprecation, what do you think?

Reply:
You are right, async: true needs to be the right one now for $.ajax({  (but it is the default in recent Jquery, what version do you have?).

We can force it, but if so, we need to put the full br initialization inside the Ajax response
Or one may trigger before the Book reader is already there. We can try/test. Does that make sense?
Who loves JS? not me!

Reply:
Something like

$.ajax({     url : "example.com",    async : true /* default is true */ }).done(function(response){     // Start BR})